hicommonwealth/commonwealth
# add optional topics to getGroups query

## Link to Issue
Closes: #5401

## Description of Changes
- Optionally adds associated topics to the payload when querying groups

## Test Plan
- CA Test
  - Set use to super admin
  - Create a group with `POST http://localhost:8080/api/groups` with body:
```json
{
    "author_chain": "<CHAIN>",
    "chain": "<CHAIN>",
    "address": "<ADDRESS>",
    "jwt": "<JWT>",
    "metadata": {
        "name": "Hello",
        "description": "Dope group"
    },
    "requirements": [],
    "topics": [<TOPIC_ID>]
}
```
  - Make request to `PUT http://localhost:8080/api/refresh-membership` to refresh memberships, with similar body (`author_chain, chain, address, jwt`)
  - Make request to `GET http://localhost:8080/api/groups?include_members=true&include_topics=true&chain_id=<CHAIN>`– confirm that it includes both memberships and associated topics in the payload

## Deployment Plan
N/A

## Other Considerations
N/A
